lib/php/Converter.php

```diff
@@ -600,11 +602,25 @@ private function convertTo01(array $data)
 
         //missing hour in timezone offset
         if (isset($data['content']['operatingsystem']) && isset($data['content']['operatingsystem']['timezone'])) {
-            if (preg_match('/^[+-][0-9]{2}$/', $data['content']['operatingsystem']['timezone']['offset'])) {
-                $data['content']['operatingsystem']['timezone']['offset'] .= '00';
+            $timezone = $data['content']['operatingsystem']['timezone'];
```

Comment:
Unless I am wrong, `$timezone` is just a value copy here, so updated its value will have no effect on `$data`
```suggestion
            $timezone = &$data['content']['operatingsystem']['timezone'];
```

Reply:
Yes, I saw that while testing on my side